src/Raven.Server/Documents/TimeSeries/TimeSeriesRollups.cs

```diff
@@ -529,52 +485,130 @@ protected override long ExecuteCmd(DocumentsOperationContext context)
                             AlertType.RollupExceedNumberOfValues, NotificationSeverity.Warning, $"{item.DocId}/{item.Name}", new ExceptionDetails(e));
 
                         context.DocumentDatabase.NotificationCenter.Add(alert);
```

Comment:
Can this cause us to spam the user with alerts?

Reply:
it will generate a message per (problematic) ts

Reply:
That can be a lot of them, no?

Reply:
So how to notify to the user about the issue? 
Also I think there is an open issue in the studio to combine alerts by the type.

Reply:
We can use the key on the "Collection/TimeSeries", no? That would reduce the number of unique matches.

Reply:
Without the docId, since we assume that all ts in the same collection will have the same structure? 
I'm fine with that :)

Reply:
yes

Reply:
done